src/components/common/YourCompany/CorrectName/CorrectName.vue

```diff
@@ -152,6 +156,10 @@ export default class CorrectName extends Vue {
     return (this.correctNameChoices.length === 1)
   }
 
+  get nameChangeAction (): string {
+    return this.isAlterationFiling ? 'alter' : 'correct'
+  }
```

Comment:
Would it be a lot more work to pass in a prop instead?

Eventually it would be good to use the [shared](https://github.com/bcgov/bcrs-shared-components/tree/main/src/components/correct-name) version of this component and not using the store here would be good.

Reply:
Oh, I should have been more clear. I was thinking you'd pass a prop for nameChangeAction :)

Reply:
PS https://github.com/bcgov/bcrs-shared-components/blob/1cac74c93fdf3d8c02cbcfe232b797d3fdb6d863/src/components/correct-name/CorrectName.vue#L124